contact.html

```diff
@@ -0,0 +1,108 @@
+              <ul id="form-ul">
+                <li>
+                  <input
+                    type="text"
+                    name="username"
+                    id="user-name"
+                    placeholder="Nume"
+                  />
+                </li>
+                <li>
+                  <input
+                    type="email"
+                    name="useremail"
+                    id="user-email"
+                    placeholder="Email"
+                  />
+                </li>
+                <li>
+                  <input
+                    type="text"
+                    name="subject"
+                    id="subject"
+                    placeholder="Subiect"
+                  />
+                </li>
+                <li>
+                  <textarea
+                    name="user-message"
+                    id="usermessage"
+                    rows="5"
+                    placeholder="Mesaj"
+                  ></textarea>
+                </li>
+              </ul>
+
+              <div class="btn-form">
+                <button type="submit" class="btn-filled-form">
+                  Trimite mesaj
+                </button>
+              </div>
+          </form>
+        </div>
+        <div id="map">
+          <img src="img/imag4.png" alt="map" />
+        </div>
+      </section>
+
+      <section class="pictures">
+        <a href="nusa-penida.html" class="pic1"><img src="img/nusa_penida3.jpg" alt="" ></a> 
+        <a href="petra.html" class="pic2"><img src="img/petra3.jpg" alt="" ></a>
+        <a href="lacul-sorapiss.html" class="pic3"><img src="img/lacul_sorapiss.jpg" alt=""></a>
+        <a href="insula-siargao.html" class="pic4"> <img src="img/insula_siargao1.jpg" alt="" ></a>
+        <a href="santorini.html" class="pic5"><img src="img/santorini4.jpg" alt=""></a>
+        <a href="petra.html" class="pic6"><img src="img/petra.jpg" alt=""></a> 
+        <a href="mattehorn.html"  class="pic7"> <img src="img/mattehorn_2.webp" alt=""></a>
+        <a href="insula-siargao.html" class="pic8"> <img src="img/insula_siargao3.jpg" alt=""></a>
+        <a href="mattehorn.html" class="pic9"> <img src="img/mattehorn.jpg" alt=""></a>
+        <a href="nusa-penida.html" class="pic10"><img src="img/nusa_penida.jpg" alt=""></a>
+        <a href="santorini.html" class="pic11"><img src="img/santorini2.jpg" alt=""></a>
+        <a href="lacul-sorapiss.html" class="pic12"><img src="img/lacul_sorapiss3.jpg" alt=""></a>
+      </section>
+
+
+      <footer>
+        <a href="https://www.instagram.com/"
+          ><img src="icons/instagram.png" alt="instagram" class="icon"
+        /></a>
+        <a href="https://facebook.com"
+          ><img src="icons/facebook.png" alt="facebook" class="icon"
+        /></a>
+        <a href="https://tiktok.com"
+          ><img src="icons/titkok.png" alt="tiktok" class="icon"
+        /></a>
```

Comment:
Mici probleme de formatare aici.
```suggestion
        <a href="https://tiktok.com">
          <img src="icons/titkok.png" alt="tiktok" class="icon" />
        </a>
```